scilpy/samplingscheme/save_scheme.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import numpy as np
+
+from scilpy.utils.filenames import split_name_with_nii
+
+
+def save_scheme_caru(points, shell_idx, filename):
+    """
+    Save gradient table (Caruyer format)
+
+    Parameters
+    ----------
+    points: numpy.array
+        bvecs normalized to 1.
+    shell_idx: numpy.array
+        Shell index for bvecs in points.
+    filename: output file name
+    ------
+    """
+
+    fullfilename, ext = split_name_with_nii(filename)
+    fullfilename = fullfilename + '.caru'
+    f = open(fullfilename, 'w')
```

Comment:
Maybe could be put in another pull request, but, for all file writting operations, we should use the managers for files from python (with open(...) as f)

Reply:
It could be nice to have this in this PR.